napari/components/_layer_slicer.py

```diff
@@ -43,9 +46,68 @@ def __init__(self):
         """
         self.events = EmitterGroup(source=self, ready=Event)
         self._executor: Executor = ThreadPoolExecutor(max_workers=1)
+        self._force_sync = False
         self._layers_to_task: Dict[Tuple[Layer], Future] = {}
         self._lock_layers_to_task = RLock()
 
+    @contextmanager
+    def force_sync(self):
+        """Context manager to allow a temporary forced synchronous slice.
+        This method only holds the _force_sync variable as True while the
+        manager is open, then resets it back to False after the manager is
+        closed.
```

Comment:
I would remove this docstring - it just describes the current implementation in a more verbose and less readable way.

Reply:
Haha. I got pinged for lack of doc strings on the last PR so maybe I swung the other way 😆